app/assets/javascripts/discourse/app/templates/preferences/notifications.hbs

```diff
@@ -41,6 +41,11 @@
 
 <UserNotificationSchedule @model={{this.model}} />
 
+<PreferenceCheckbox
```

Comment:
I think this looks (mistakenly) like it is related to the **Notification Schedule** heading:

![image](https://github.com/discourse/discourse/assets/920448/4ddb03b4-971f-4d97-af50-3662a2e6377a)

Also IMO it makes much more sense to have this in the Tracking part of user prefs eg https://meta.discourse.org/u/martin/preferences/tracking

![image](https://github.com/discourse/discourse/assets/920448/160b74e0-2f79-46c8-8eac-9a18b9d88630)

Reply:
Ya I was going to give it a heading or something so it didn't look like it was part of the Notification Schedule, but yes, the tracking page looks like a better home, I'll move it over there. Thanks!

Reply:
How does this look over on the tracking page?

![CleanShot 2024-03-07 at 16 48 58@2x](https://github.com/discourse/discourse/assets/1490496/9e3b0ece-9c2e-4bfb-be8c-3dd5de62b3fd)